packages/analytics/utils/elastic_analytics_utils/src/build_headers.test.ts

```diff
@@ -6,11 +6,11 @@
  * Side Public License, v 1.
  */
 
-import { buildHeaders } from './build_headers';
+import { buildShipperHeaders } from './build_headers';
```

Comment:
nit on the naming: it's specific to the `elastic_v3` shipper. Should it be part of the name of the helper?

Reply:
And following that line of thought... since the shipper is instantiated by the `telemetry` plugin... should we move these helpers to the plugin?

Reply:
Makes sense will do both :+1: